pkg/pillar/agentlog/agentlog.go

```diff
@@ -341,14 +317,29 @@ func roundToMb(b uint64) uint64 {
 	return mb
 }
 
-func Init(agentName string, curpart string) error {
+func Init(agentName string, curpart string) {
 	if curpart != "" {
 		zboot.SetCurpart(curpart)
 	}
```

Comment:
Heads up that we have a separate PR getting rid of curpart.

Reply:
Either we make then into one PR or ...
They touch the same lines of code in all the agents.
(Which is a reason for everybody to keep an eye at the PRs in flight to avoid collisions like this.)

Reply:
I can wait for the #794 to be passed through and then change this PR accordingly, or we can do it as one PR. Either is fine. What do you think is best?

Reply:
Yeah.. LEts get Gopi's Change in..

Reply:
Thinking more.. He can just remove curpart as an argument for Init - that should be sufficient for this PR?

Reply:
There is also the change to remove the Impl function, which is only called in one place now.